docs/install_raspbian.md

```diff
@@ -1,154 +1,217 @@
+
+We will be running the OVOS environment as a regular user, and we want OVOS to start at boot time, so we want to autologin.
+
+Enter the `System Options` page.
+
+Enter the `Boot / Autologin` page.
+
+- Use the second option in the menu, `Console Autologin`.
+  - This enables OVOS to start up automatically at boot time.
+
+Now we will enable a few interface options.  This will allow us to access our device from a `ssh shell` and prep the PI for other devices that may be used.  Some microphone hats require SPI, or I2C (Respeaker, AIY-Voicebonnet, etc).
+
+Go back to the main menu and enter the `Interface Options` page.
+
+- Enable SPI, and I2C.
+
+Go back to the main menu and enter the `Localisation Options` page.
+
+- Configure Locale, Timezone, WLAN Country.
+
+Now the device setup is done.  Exit raspi-config and reboot.
 
 `sudo reboot now`
 
+### Step 2: ssh to your OVOS device
+
 <strong>*** From this point on, you should be able to access your device from any SSH terminal. *** </strong>
 
 For guide for how to do this, see [raspberrypi documentation remote-access](https://www.raspberrypi.com/documentation/computers/remote-access.html)
 
-From a linux machine, open a terminal and enter the command `ssh ovos@<your-remembered-IP-address>`.  There will be a warning making sure you want to connect to this device.  Enter yes, and when asked, enter the password for ovos that you made earlier in the setup. `ovos`
+From a linux machine, open a terminal and enter the command `ssh ovos@<your-remembered-IP-address>` or `ssh ovos@<your-hostname>`.  There will be a warning making sure you want to connect to this device.  Enter yes, and when asked, enter the password for ovos that you made earlier in the setup.
+`ovos`
 
-First you need to make sure your system is up to date.  It should be close as you just installed a new image.
+As a final configuration step, make sure your system is up to date.
 
 `sudo apt -y update && sudo apt -y upgrade`
 
 We should be done with the basic setup now.  You should have a running RaspberryPiOS device with the user `ovos`
 
-## Step 3: Install OVOS-CORE
+### Step 3: Install ovos-core
 
-There are some recommendations to use a venv for ovos. This guide DOES NOT do that.  The ovos headless stack on a RPi3 is about all it can handle.  It is assumed that this is a dedicated ovos device, therefore no venv is required.
+There are some recommendations to use a venv for OVOS. This guide DOES NOT do that.  The OVOS headless stack on a RPi3 is about all it can handle.  It is assumed that this is a dedicated OVOS device, therefore no venv is required.
 
-There are a few packages required for ovos, so we will install those first
+We will be cloning code from a git repository, so before starting we need to install git.
 
-`sudo apt install build-essential python3-dev python3-pip swig libssl-dev libfann-dev portaudio19-dev libpulse-dev cmake git libncurses-dev pulseaudio-utils`
```

Comment:
Are these all included on the Raspbian image from OVOS?

Reply:
yes, the image includes all of those packages

Reply:
and it is also in the `manual_user_install.sh` script.  Should not need it here